README.md

```diff
@@ -15,6 +15,16 @@ The `packages` directory contains all the individual components. Let's talk abou
 - `docs.mdx`: documentation for the component, see [swingset docs](https://github.com/hashicorp/swingset#usage) for more details on the format
 - `props.js`: information about the component's props, see [swingset docs](https://github.com/hashicorp/swingset#props) for more details on the format
 
+## Adding package dependencies
```

Comment:
Added this note to clarify the preferred mode for installing package dependencies. Also added a note on wanting to not have `package-lock.json` files in package subfolders.